.gitattributes

```diff
@@ -0,0 +1 @@
+vendor/* linguist-generated=true
```

Comment:
Added this to have GitHub collapse all the vendor files by default when reviewing 😅